.github/workflows/cache.yml

```
@@ -0,0 +1,20 @@
+            - name: Cache
```

actionlint (1.7.4)

workflow is sequence node but mapping node is expected

(syntax-check)